### PR TITLE
Removes Cringe Metashield from IPC Martial Arts - Wings are obvious as fuck

### PR DESCRIPTION
**Pull request by JamieD1**
image

Wings are now on examine and if they are martial arts wings they are obvious via hacked message.

:cl:  
rscadd: Examine on Wings
tweak: IPC Martial Arts is now Obvious
/:cl:

### CODE REVIEW

**Comment by Hopekz** [comment]
Yes because the average crew member is expected to know the nuances of wing type.

Imo it'll be better to make them all valid or not valid vs this "pick and choose" type implementation.

I'm talking about it in a metagame aspect because now all of a sudden people are expected to know the difference in-charecter (at least that is what is implied by your reasoning)

**Comment by JamieD1** [comment]
> Yes because the average crew member is expected to know the nuances of wing type.
> 
> Imo it'll be better to make them all valid or not valid vs this "pick and choose" type implementation.

Im happy to remove wings from shitty ipc martial art, its the only martial art that makes a physical difference, kinda dumb imo

**Comment by Moltijoe** [comment]
> Im happy to remove wings from shitty ipc martial art, its the only martial art that makes a physical difference, kinda dumb imo

I initially didnt include them, but people begged me to because the wings are also an ultrakill reference

**Review by ynot01 — APPROVED** [review]

**Comment by Hopekz** [comment]
> Im happy to remove wings from shitty ipc martial art, its the only martial art that makes a physical difference, kinda dumb imo

Your choice on what direction to go with in order to commit to one direction but it is an inconsistency ATM

Not going to pretend the wings aren't cool af though

File: code/modules/mob/living/carbon/human/examine.dm [hunk 1]
```diff
@@ -113,6 +113,12 @@
 		if(100 to 200)
 			. += span_warning("[t_He] [t_is] twitching ever so slightly.")
 
+	if("wings" in dna.species.mutant_bodyparts)
+		var/hacked = ""
+		if(mind?.martial_art && istype(mind.martial_art, /datum/martial_art/ultra_violence))
+			hacked = ionnum() + ionnum() + " "
+		. += "[t_He] [t_has] also have a pair of [span_warning(hacked)][(dna.features["wings"])] wings on their back"
```
Comment (SomeguyManperson) [inline code/modules/mob/living/carbon/human/examine.dm]: Blood drenched wings perhaps? Hacked is something strange to say someone just sees

**Comment by ynot01** [comment]
Ultra Violence has existed this long being a meta-questionable "I am valid" sign and I think it's clear that the martial art basically encourages that playstyle, so this will turn out fine I think. This may provoke validhunters, but in a perfect world, as long as the IPC is not attacking randoms or killing in broad daylight, randoms shouldn't be interfering.

**Comment by Moltijoe** [comment]
> Ultra Violence has existed this long being a meta-questionable "I am valid" sign and I think it's clear that the martial art basically encourages that playstyle, so this will turn out fine I think. This may provoke validhunters, but in a perfect world, as long as the IPC is not attacking randoms or killing in broad daylight, randoms shouldn't be interfering.

if someone's validhunting, they probably don't care too much about meta in the first place. nice side effect is that lets IPCs get wings from other sources and not immediately be lynched

**Review by Moltijoe — APPROVED** [review]
no more trying to be stealthy with the bloodsoaked murderbone martial art

**Pull request merged**